db/migrate/20190121210000_create_recipes.rb

```diff
@@ -0,0 +1,18 @@
+# Create Recipes Table
+
+class CreateRecipes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :recipes do |t|
+      ## 
+      t.string :title,       null: false, default: ""
+      t.string :snipped,     null: false, default: ""
+      t.string :meal,        null: false, default: ""
+      t.text   :ingredients, array: true, default: []
+      t.text   :directions,  array: true, default: []
+      t.string :difficulty,  null: false, default: "Intermediate"
+      t.integer :duration,   null: false, default: 0
+      t.integer :servings,   null: false, default: 2
+      t.string  :photos,     array: true, default: []    
```

Comment:
I think it'd be good to leave off most/all of these defaults. Might be better to control that stuff in software, vs. at the database level. I also think most of these fields could be null (at least as far as the database is concerned), and we can handle validation further up the stack.